web/client/components/share/ShareCopyToClipboard.jsx

```diff
@@ -24,6 +24,7 @@ const ShareCopyToClipboard = ({
             text={shareUrl}
             tooltipId={copied ? 'share.msgCopiedUrl' : 'share.msgToCopyUrl'}
             tooltipPosition="bottom"
+            key={copied.toString()}
```

Comment:
I found that to fix the problem with a tooltip is to trigger its rerender. That's why I use the key prop associated with the copied value.